docs/user/dashboard/tsvb.asciidoc

```diff
@@ -166,3 +166,15 @@ For other types of month over month calculations, use <<timelion, *Timelion*>> o
   
 Calculating the duration between the start and end of an event is unsupported in *TSVB* because *TSVB* requires correlation between different time periods. 
 *TSVB* requires that the duration is pre-calculated.
+
+[float]
+===== How do I group on multiple fields?
+  
+Grouping with multiple fields is not natively supported in *TSVB* but you could use runtime fields to achieve this.
```

Comment:
Instead of calling out how we don't support grouping with multiple fields in TSVB, could we reword this a bit to show users how we are providing them with this feature? 

For example, could we simplify this to:

`To group with multiple fields, create runtime fields in the index pattern you are visualizing.`

I'm thinking that basic-level data analysts might not know the difference between native support in TSVB and the rest of Kibana. Also, I always get sad when I read about features that aren't natively supported, but are supported with a workaround.

Reply:
You are right!! I am changing it!